uint32 time
float64 game_time
uint32 sample_count
monodrive_msgs/VehicleState[] vehicles
